tests/php/test-amp-analytics-options.php

```diff
@@ -269,13 +275,27 @@ function ( $entries ) use ( $analytics, &$trigger_count ) {
 				$trigger_count++;
 			}
 		);
-
+		
```

Comment:
This whitespace is causing a linting error: https://travis-ci.org/ampproject/amp-wp/jobs/593274574#L777-L787